package.json

```diff
@@ -70,6 +66,7 @@
 	},
 	"peerDependencies": {
 		"react": ">= 16.8.0",
+		"react-dom": ">= 16.8.0",
```

Comment:
I decided to keep this in parity with `react`, if there was a particular reason you had `react-dom` at `16.10.2` before then let me know and I'll update both to that as a min version instead.